aiida/orm/nodes/data/code.py

```diff
@@ -131,21 +152,14 @@ def relabel(self, new_label, raise_error=True):
         if new_label.endswith(suffix):
             new_label = new_label[:-len(suffix)]
 
-        if '@' in new_label:
-            msg = "Code labels must not contain the '@' symbol"
-            if raise_error:
```

Comment:
Ok to deprecate it - in this case please put in the docstring of the function something like
```
.. deprecated:: 1.2.0
            Will be removed in `v2.0.0`. Use `try/except` instead.
```

Reply:
I'll remove the argument, it will make `label` setter complex otherwise.

Reply:
What you've done here is fine (leave the argument for the moment but don't do anything with it).
Can you still put the deprecation warning about the `raise_error` argument into the docstring of `relabel`?